<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Qygmy</string>
  </property>
  <widget class="QWidget" name="v">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <widget class="ClickableProgressBar" name="progressbar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string notr="true">Disconnected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="volume_slider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="volume_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string notr="true">50</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="current_song">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string notr="true">&lt;big&gt; &lt;/big&gt;&lt;br&gt;&lt;small&gt; &lt;/small&gt;&lt;br&gt;&lt;small&gt; &lt;/small&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="SonglistView" name="queue">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="status">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="playback_toolbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="previous"/>
   <addaction name="play"/>
   <addaction name="pause"/>
   <addaction name="stop"/>
   <addaction name="next"/>
   <addaction name="volume"/>
  </widget>
  <widget class="QToolBar" name="queue_toolbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="add"/>
   <addaction name="remove"/>
   <addaction name="clear"/>
   <addaction name="separator"/>
   <addaction name="repeat"/>
   <addaction name="shuffle"/>
   <addaction name="single"/>
   <addaction name="consume"/>
   <addaction name="separator"/>
   <addaction name="settings"/>
  </widget>
  <action name="previous">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="play">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="stop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="next">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next Song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="add">
   <property name="text">
    <string>Add...</string>
   </property>
   <property name="toolTip">
    <string>Add to playlist...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="remove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="repeat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
  <action name="shuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="toolTip">
    <string>Settings...</string>
   </property>
  </action>
  <action name="volume">
   <property name="text">
    <string>Volume</string>
   </property>
  </action>
  <action name="clear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear playlist</string>
   </property>
  </action>
  <action name="disconnect">
   <property name="text">
    <string>Disconnect from MPD</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="connect">
   <property name="text">
    <string>Connect to MPD</string>
   </property>
  </action>
  <action name="updatedb">
   <property name="text">
    <string>Update the database</string>
   </property>
  </action>
  <action name="single">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Single track mode</string>
   </property>
   <property name="iconText">
    <string>Single</string>
   </property>
  </action>
  <action name="consume">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove songs after playback</string>
   </property>
   <property name="iconText">
    <string>Consume</string>
   </property>
  </action>
  <action name="pause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Save the current playlist...</string>
   </property>
   <property name="toolTip">
    <string>Save the current playlist</string>
   </property>
  </action>
  <action name="details">
   <property name="text">
    <string>Song details...</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="randomize">
   <property name="text">
    <string>Randomize the playlist</string>
   </property>
  </action>
  <action name="statistics">
   <property name="text">
    <string>MPD statistics...</string>
   </property>
  </action>
  <action name="highprio">
   <property name="text">
    <string>High priority</string>
   </property>
   <property name="toolTip">
    <string>Set high priority for selected songs</string>
   </property>
  </action>
  <action name="normprio">
   <property name="text">
    <string>Normal priority</string>
   </property>
   <property name="toolTip">
    <string>Set normal priority for selected songs</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SonglistView</class>
   <extends>QTreeView</extends>
   <header>..uiutils</header>
  </customwidget>
  <customwidget>
   <class>ClickableProgressBar</class>
   <extends>QProgressBar</extends>
   <header>..uiutils</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
